frontend/packages/dev-console/integration-tests/features/Knative/connector-Event-Sources-ODC-4230.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Connector Event Sources
```

Comment:
would be good to use `CamelK connector Event Sources` for more clarity as these are CamelK connectors

Reply:
Updated.